raiden/utils/cli.py

```diff
@@ -0,0 +1,205 @@
+from ipaddress import IPv4Address, AddressValueError
+from itertools import groupby
+from typing import Callable, List
+
+import click
+from click._compat import term_len
+from click.formatting import iter_rows, measure_table, wrap_text
+
+from raiden.utils import address_decoder
+
+
+class HelpFormatter(click.HelpFormatter):
```

Comment:
Hmm ... why are you making a special help formatter here? What are you trying to solve and what does it have to do with the Matrix protocol?

Reply:
It doesn't have anything to do with matrix per-se but I had this laying around in 3/4 finished form for a while and since I was adding yet another cli option I thought it was a good oppotunity.

What this is used for is to split the options into logical groups. The output now looks like this: https://gist.github.com/ulope/421fde0f37be1274159c6bbb41186c03